Assets/MixedRealityToolkit/Utilities/MaintainBorderLightWidth.cs

```diff
@@ -42,7 +42,7 @@ private void Start()
 
         private void LateUpdate()
         {
-            if (targetRenderer != null && prevScale != transform.lossyScale)
+            if (prevScale != transform.lossyScale && targetRenderer != null)
```

Comment:
Any reason for this change?

Reply:
comparing unity object is more expensive than comparing vector3s. Unity has to cross native/managed threshold and check both native pointer and managed pointer

If the first condition fails, the compiler doesn't process the second condition

Reply:
Oh wow! I didn't know that. Thanks!